models/intel/instance-segmentation-security-0002/description/instance-segmentation-security-0002.md

```diff
@@ -0,0 +1,45 @@
+# instance-segmentation-security-0002
+
+## Use Case and High-Level Description
+
+This model is an instance segmentation network for 80 classes of objects.
+It is a Mask R-CNN with ResNet50 backbone, FPN, RPN, detection and
+segmentation heads.
+
+## Example
+
+![](./instance-segmentation-security-0002.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| MS COCO val2017 box AP          | 40.8%                                     |
+| MS COCO val2017 mask AP         | 36.9%                                     |
+| Max objects to detect           | 1000                                      |
```

Comment:
This 1000 does not match with shape of network outputs described below.

Reply:
Fixed to the output shape.